package.json

```
@@ -34,6 +46,19 @@
     "clean": "gatsby clean",
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "jest",
+      "post-push": "npm run build"
+    }
+  },
+  "lint-staged": {
+    "./src/*.{js,jsx,ts,tsx}": [
```

Modificar esta parte par que solo se admita archivos {ts, .tsx}
```suggestion
    "./src/*.{ts,tsx}": [
```